docs/CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ nav_order: 5
 
      *Mike Munroe*
 
+* Add support for csp nonces inside of components
```

Comment:
```suggestion
* Add support for CSP nonces inside of components.
```

Reply:
I will fix that when I write the tests tonight thank you